### PR TITLE
improve: Update sdk-v2 and contracts-v2 to non-beta versions

### PR DESCRIPTION
**Pull request by nicholaspai**
Includes a fix to the `hardhat.config` file to deal with imported `hardhat` objects from updated packages

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -4,7 +4,10 @@ import { HardhatUserConfig } from "hardhat/config";
 import { getNodeUrl, getMnemonic } from "@uma/common";
 
 // Custom tasks to add to HRE.
-require("./tasks");
+// @dev: Temporarily commenting out tasks to minimize amount of files imported and executed at compile time
```
Comment (james-a-morris) [inline hardhat.config.ts:7]: Should we add a `// FIXME: ` or `// TODO: ` or include this in a backlogged Linear ticket?

**Review by james-a-morris — APPROVED** [review]
LGTM - have a nit

File: hardhat.config.ts (hunk 1 above)
Comment (nicholaspai) [reply hardhat.config.ts:7]: Great call, added

**Review by pxrl — APPROVED** [review]
LGTM

**Pull request merged**